### ISSUE TITLE
Can't attach any media files?!

### ISSUE THREAD

**Issue by whowantsmybigdata** [ChatSecure/ChatSecure-iOS#1179]
As Monal can't handle OMEMO in Group-Chats and while sending media, I just installed ChatSecure v5.0 on my fathers iPhone 7 running iOS 13.3. 

Group-Chats are working (great!) but suprisingly there is no obvious way to attach any media at all. 
The interface looks like the one in #1148, so there is no attachment-icon, camera-icon or whatsoever only the typing-texts area and the send-button. 
Sharing via CS in other apps (as Photos e.g.) isn't possible as well but I guess this is normal, right?!

I didn't find anything related to that and couldn't solve it via uninstall/reinstall...
Server-sided (self-hosted ejabbered server) everything seems to be working aswell, as Monal and Conversations are both working normally on the same and other devices with attachable media-files (besides the mentioned app-side probs with monal) via http-upload.

Thanks a lot for the effort of finally putting up a promising XMPP-Omemo-capable client for iOS!

**Comment by chrisballinger** [comment]
If your server doesn't advertise support for XEP-0363 it won't work. Try checking the Server Information tab in your Account settings.

**Comment by whowantsmybigdata** [comment]
Yes it does support XEP-0363! on conversations and monal http-upload (which is XEP-0363 right?!) is working.

**Comment by whowantsmybigdata** [comment]
any other ideas?

**Comment by zerobytes999** [comment]
Hello. I can confirm the same issue.

Client 1: Android - conversations
Client 2: iOS - chatsecure.
Server: Openfire v4.5

Server advertising XEP-0363 and it works well on coverstions.

It shows in chatsecure XEP-0363 is available(green tick) but it is not enabling media controls (attach file, take picture... etc)

**Comment by Mythra** [comment]
I know this is a very old issue, but curious if anyone ever found an answer to this. We've got a server where XEP-363 has a green tick, we've even set a max upload size of files (incase: https://github.com/ChatSecure/ChatSecure-iOS/issues/1241 was being hit), and yet multiple of us don't seem to have a camera icon. Curious what else we could try.
